factory/factory.go

```diff
@@ -0,0 +1,23 @@
+package factory
+
+import (
+	"github.com/bitcoin-sv/go-broadcast-client/broadcast"
+	"github.com/bitcoin-sv/go-broadcast-client/broadcast/arc"
+	"github.com/bitcoin-sv/go-broadcast-client/config"
+)
+
+func NewBroadcastClient(factories ...broadcast.BroadcastFactory) broadcast.Broadcaster {
+	return broadcast.NewCompositeBroadcaster(config.DefaultStrategy, factories...)
```

Comment:
I think that this is the only point I have doubts about.

If we're creating CompositeBroadcaster even for single BroadcastFactory, then what is the reason for having this CompositeBroadcaster separated. And also what is the reason for WithMultiminers as it is doing exactly the same, and actually it will be even less performant because all broadcasters will be wrapped 2 times into a CompositeBroadcaster

Reply:
Done